### ISSUE TITLE
DocCommentSpacing does not handle @inheritdoc properly

### ISSUE THREAD

**Issue by guvra** [slevomat/coding-standard#1436]
Hello,

DocCommentSpacing sniff is treating @inheritdoc like a standard annotation, which makes it impossible to use it properly in some situations.

Example config:

```xml
    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing">
        <properties>
            <property name="linesCountBeforeFirstContent" value="0"/>
            <property name="linesCountAfterLastContent" value="0"/>
            <property name="linesCountBetweenDescriptionAndAnnotations" value="1"/>
            <property name="linesCountBetweenDifferentAnnotationsTypes" value="0"/>
        </properties>
    </rule>
```

With `{@inheritdoc}`:

```php
    /**
     * {@inheritdoc}
     *
     * @throws RuntimeException
     */
    public function example();
```

✓ No error (as expected).

But with `@inheritdoc`:

```php
    /**
     * @inheritdoc
     *
     * @throws RuntimeException
     */
    public function example();
```

Expected result: no error
Actual result: Slevomat reports the following error:

```
Expected 0 lines between different annotations types, found 1. (SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectLinesCountBetweenDifferentAnnotationsTypes)
```

If the sniff is configured to use annotation groups, it also triggers errors:

```
Incorrect order of annotations groups. (SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectOrderOfAnnotationsGroup)
Expected 0 lines between annotations groups, found 1. (SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectLinesCountBetweenAnnotationsGroups)
```

**Comment by kukulich** [comment]
> If the sniff is configured to use annotation groups, it also triggers errors:

Can you please post your configuration for this case?
